g2p/convert.py

```diff
@@ -36,18 +43,28 @@ def run(self):
 
 
 class G2POutputToBlissLexiconJob(Job):
-    def __init__(self, iv_lexicon, g2p_output, merge=True):
-        self.iv_lexicon = iv_lexicon
-        self.g2p_output = g2p_output
+    """
+    Convert a g2p applied word list file into a lexicon
+    """
+
+    def __init__(self, iv_bliss_lexicon, g2p_output_lexicon, merge=True):
+        """
+        :param Path iv_bliss_lexicon: bliss lexicon as reference for the phoneme inventory
+        :param Path g2p_output_lexicon: from ApplyG2PModelJob.out_g2p_lexicon
+        :param bool merge: merge the g2p lexicon into the iv_bliss_lexicon instead of
+            only taking the phoneme inventory
+        """
+        self.iv_bliss_lexicon = iv_bliss_lexicon
+        self.g2p_output_lexicon = g2p_output_lexicon
```

Comment:
better: g2p_lexicon

Reply:
Is this in the exact same format as the input to the G2P training? If yes then this makes sense to change. I was not sure if they might look different.

Reply:
Yes, they are the same